source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -609,53 +613,63 @@ namespace AdaptiveCards
                 AdaptiveActionSetRenderer();
             };
 
-            [default_interface] runtimeclass AdaptiveOpenUrlActionRenderer : IAdaptiveElementRenderer
+            [default_interface] runtimeclass AdaptiveOpenUrlActionRenderer : IAdaptiveActionRenderer
```

Comment:
>IAdaptiveActionRenderer [](http://example.com/codeflow?start=77&length=23)

@paulcam206 It would be a break change. If we really need this, I would like to split it to two PRs:
one PR for migration, and another PR for interface and api change

Reply:
Overwriting **ActionRenderers** never worked in **UWP** because in the **Render** function they expect an **Action** but in the IDL they inherit from **ElementRenderer**. Trying to instantiate them in **UWP** crashes the app.
This fixes it.